lib/rspec/core/did_you_mean/did_you_mean.rb

```diff
@@ -0,0 +1,9 @@
+module RSpec
+  module Core
+    # Intermediate auto-loading file
+    module DidYouMean
+      autoload :Proximity, 'rspec/core/did_you_mean/proximity'
+      autoload :Suggestions, 'rspec/core/did_you_mean/suggestions'
+    end
+  end
+end
```

Comment:
Can we ditch the two child files and have it all in here please.

Reply:
Done. I also ditched the directory given there is now only one file.